### ISSUE TITLE
Doc should inform user that the `--ignore` option takes a list of comma-separated directories as input

### ISSUE THREAD

**Issue by MapleCCC** [bndr/pipreqs#213]
It's not shown in the doc that the `--ignore` option takes a list of comma-separted directories as input. After failing with space-separated directoreis, I need to dwell into the source code to figure this out. It would be better if the doc could inform user about this behaviour. We don't want to make user surprised.

**Comment by rahul-kumi** [comment]
When invoking the help for `pipreqs` through the following command
```sh
$pipreqs --help
```
We can notice the following statement that clearly states that the directories passed to the `--ignore` option are to be separated by commas.
```sh
--ignore <dirs>...    Ignore extra directories, each separated by a comma.
```

This is not clarified in `README.rst`.

**Comment by alan-barzilay** [comment]
closed by #254
